### ISSUE TITLE
CARD-NEW-TOP10-RESEARCH-B0-MATRIX-AUTHORITY-AND-SEARCH-DESIGN

### ISSUE THREAD

**Issue by bluemaple18-home** [bluemaple18-home/NEW-TOP10#13]
## Current state

`READY_FOR_READ_ONLY_RESEARCH / NO IMPLEMENTATION / NO QUEUE WRITE`

- Successor to completed Card A: #1.
- NEW-TOP10 baseline: `2b9eccda11433016261c529ad1f94352bcfcd6d5`.
- AI Core baseline: `26eb42f8e401807d3dbd6030171b9dfeb58207fb`.
- May run in parallel with C0.
- B1 and all runtime/control changes remain **NOT ADMITTED**.

## Product intent

NEW-TOP10 最終要形成一座依市場盤況運作的長期研究工廠：

```text
股票專屬合法矩陣
→ 每天依 regime、coverage、learning、failure 與研究預算挑下一批組合
→ 回測 / replay / forward shadow
→ 成功、合法負向、失敗、未知都留下可稽核證據
→ 依證據做粗到細微調、複驗、反證與交互作用研究
→ 每種市場盤況產出有證據、有效期、替代方案與 fallback 的 RegimePolicyBundle
```

本卡只負責證明「股票自己的矩陣是什麼」以及選擇何種研究搜尋方法；不得把 Trace V2、Optuna 或其他 donor 的產品規則直接當成 NEW-TOP10 authority。

## Root question

NEW-TOP10 能否以一份可稽核的 Matrix Definition，精確描述完整股票研究組合宇宙、合法限制、組合數、鄰接與評估成本，並據此裁決哪些區域應完整掃描、哪些區域應採 evidence-driven adaptive research？

## Current repo facts that must be reconciled

Must inspect at the pinned baseline:

- `config/research_parameter_catalog.json`
- `config/regime_research_contract.json`
- `app/research/parameter_catalog.py`
- `app/research/parameter_learning.py`
- `app/research/adaptive_shadow_queue.py`
- `scripts/run_backtest_strategy_matrix.py`
- `app/research/contracts.py`
- #1 Card A authority contract

Current committed facts include:

- only four dimensions are presently proven executable: `horizon`, `stop_loss_pct`, `take_profit_pct`, `max_group_exposure`;
- current formal executable count is `720`;
- `regime_gate`, `risk_guard`, `entry_filter` remain contract-dependent / coverage-only;
- the claimed full two-million-scale stock matrix has not yet been proven from a canonical dimension source;
- existing `parameter_learning` and `adaptive_shadow_queue` are reusable seams, not authority for the final matrix or production queue.

## Non-equivalence with Trace V2

Trace may donate only generic constrained-combination concepts after a pinned source is supplied:

- discrete legal combination generation;
- exact combination counting;
- deterministic ordering / rank-unrank;
- chunked vectorized evaluation;
- no per-candidate I/O;
- benchmark and winner/audit evidence patterns.

Do **not** assume any Trace-specific product rule applies to stocks:

- no assumed 15 strategies;
- no assumed sum-to-100 allocation vector;
- no assumed 10 grids or `1,961,256` candidates;
- no 15% mandatory floor;
- no CPC/CPM scoring;
- no two-million candidate cap unless NEW-TOP10 independently adopts and justifies one;
- no immediate winner write-back.

If a canonical Trace source cannot be pinned, classify it `UNPINNED_CROSS_PROJECT_DONOR` and continue without it.

### 1. Matrix authority

Identify every real stock research dimension and its sole authoring source. Classify each dimension as:

- Cartesian numeric / ordinal;
- categorical;
- constrained allocation;
- conditional;
- structural choice;
- dynamic / context-resolved;
- coverage-only;
- executable;
- unsupported or unknown.

Do not create a second allowed-values authority beside the Parameter Catalog.

### 2. Matrix Definition and exact count

Draft a versioned `research-matrix-definition.v1` or equivalent containing:

- referenced catalog/component versions and hashes;
- dimensions selected for this matrix;
- cross-dimension constraints;
- invalid-combination rules;
- deterministic generation policy;
- raw Cartesian count;
- invalid/pruned count;
- exact legal combination count;
- count receipt and reproducibility evidence;
- rank/unrank and chunking feasibility.

If the two-million-scale claim cannot be reproduced, report the exact proven count and the missing authority; do not force the result to resemble Trace.

### 3. Evaluation separability and cost

Classify candidate evaluation paths:

- `E1`: pure vector/math evaluation;
- `E2`: reusable intermediate paths/components;
- `E3`: full path-dependent replay/backtest;
- `E4`: forward-shadow-only validation.

Measure or bound:

- candidate/sec;
- wall time per evaluation class;
- CPU, peak memory and I/O;
- reusable intermediate outputs;
- regime multiplier;
- exact-scan feasibility;
- cost of complete versus staged evaluation.

Existing code may be executed in an isolated temporary output area for benchmark evidence. No tracked code/config/runtime/production mutation is allowed.

### 4. Evidence-driven daily research design

Define, without implementing, candidate families such as:

- coverage fill;
- local refinement;
- boundary expansion;
- replication across independent regime episodes / lineage;
- interaction resolution;
- negative control / challenge;
- failure diagnosis;
- rare-regime reserve.

Specify how coarse-to-fine refinement works for each real dimension type. Pairwise weight transfer is allowed only if the stock matrix actually contains a constrained allocation dimension.

### 5. Outcome and evidence semantics

Keep separate:

- execution status: succeeded / failed / cancelled / timed out / orphaned / rejected-before-execution;
- research outcome: positive / valid-negative / inconclusive / unstable / overfit-risk / regime-sparse;
- evidence eligibility: adaptive-eligible / diagnostic-only / sealed-validation-only / invalid-lineage / unsupported.

Execution failure must never become a negative strategy result.

### 6. Search/admission decision

Decide, by measured evidence, where to use:

- deterministic full enumeration;
- stratified / Sobol / Latin-hypercube seeding;
- local neighborhood refinement;
- racing / successive-halving / multi-fidelity;
- optional optimizer adapter;
- no-go / insufficient-evidence.

Do not select an optimizer merely because the space is large.

### 7. Final research output

Draft the minimum `RegimePolicyBundle` contract:

- regime identity/version;
- primary combination;
- robust alternatives;
- evidence and independent episode coverage;
- validation / sealed OOS / forward-shadow status;
- overfit and sensitivity flags;
- validity window / expiry;
- fallback bundle;
- lifecycle state.

This draft has no production authority.

## Prior art and open-source sources

Use official docs/repositories; pin exact commit/tag/version and license in the output.

### Combination / sampling

- SciPy QMC / Sobol / LatinHypercube: https://github.com/scipy/scipy
- SciPy QMC docs: https://docs.scipy.org/doc/scipy/reference/stats.qmc.html

### Suggestion / execution separation

- Optuna: https://github.com/optuna/optuna
- Optuna Ask-and-Tell: https://optuna.readthedocs.io/en/stable/tutorial/20_recipes/009_ask_and_tell.html
- Google OSS Vizier: https://github.com/google/vizier
- Ray Tune: https://github.com/ray-project/ray

### Racing / multi-fidelity / optimization

- SMAC3: https://github.com/automl/SMAC3
- DEHB: https://github.com/automl/DEHB
- Ax: https://github.com/facebook/Ax
- BoTorch: https://github.com/pytorch/botorch

### Offline-policy admissibility only

- Open Bandit Pipeline: https://github.com/st-tech/zr-obp

### Financial overfit / multiple testing

- Deflated Sharpe Ratio: https://doi.org/10.3905/jpm.2014.40.5.094
- Probability of Backtest Overfitting: https://papers.ssrn.com/sol3/papers.cfm?abstract_id=2326253

### Supplemental evidence semantics

- Open Market Intelligence: https://github.com/lulu930128/open-market-intelligence

## Reuse decision taxonomy

For every donor/component, classify:

```text
ALREADY_EXISTS
USE_AS_IS
CONFIGURE
WRAP
ADAPT
COPY_CODE
CUSTOM_REQUIRED
RESEARCH_ONLY
DEFER
REJECT
```

Prefer `EXTEND_EXISTING > ADD_SUBSYSTEM`. Donor Study DB, optimizer lifecycle, queue, scheduler or runtime must not become NEW-TOP10 authority.

## Required outputs

Write a docs/evidence bundle with at least:

```text
01-current-matrix-authority-map.md
02-stock-matrix-definition-and-exact-count.md
03-dimension-and-constraint-taxonomy.md
04-evaluation-separability-and-cost-audit.md
05-full-scan-vs-adaptive-search-decision.md
06-daily-research-and-refinement-policy.md
07-outcome-evidence-and-overfit-guards.md
08-prior-art-and-open-source-reuse-matrix.md
09-regime-policy-bundle-contract-draft.md
10-b1-admission-recommendation.md
```

Each material claim must record:

```text
claim
classification
source repo
source SHA/version
file/official URL + section
observed_at
confidence
conflict_with
implication
open_question
```

## Hard constraints

- no runtime, schema, database, runner, queue, scheduler, model, backtest-math or production modification;
- no B1/B2/B3/B4 implementation;
- no optimizer dependency installation;
- no assumption that the stock matrix equals Trace;
- no assumption that a high historical score is promotable;
- no queue/execution authority in any Decision Projection;
- no unpinned donor conclusion presented as current fact;
- no use of sealed/ambiguous lineage as adaptive evidence.

## Stop / continue rules

Continue with `UNKNOWN`, `UNPINNED_RUNTIME_ARTIFACT`, or `UNPINNED_CROSS_PROJECT_DONOR` for local evidence gaps.

Stop the overall card only if:

- governing matrix authority cannot be identified because committed authorities conflict;
- identity grain cannot be determined;
- exact count requires inventing missing dimensions/constraints;
- a runtime mutation is required to answer the research question;
- research begins to touch production or sealed evidence improperly.

## Acceptance

B0 is accepted only if:

1. The real stock matrix dimensions and constraints are traced to sole authorities.
2. The legal combination count is reproduced exactly or explicitly remains unproven with a precise missing-source report.
3. Trace-specific rules are cleanly separated from reusable combination-kernel concepts.
4. Evaluation classes and measured/bounded costs justify full scan, adaptive search, or a hybrid by region.
5. Daily candidate families and coarse-to-fine refinement are evidence-driven and dimension-aware.
6. Execution failure, valid negative research and evidence ineligibility remain distinct.
7. Multiple-testing / overfit guards are included before any finalist/promotion concept.
8. A minimal RegimePolicyBundle draft exists without production authority.
9. Reuse decisions cover existing NEW-TOP10 seams and pinned open-source donors.
10. The only next-step verdict is a bounded B1 admission recommendation; no implementation is automatically admitted.

**Comment by bluemaple18-home** [comment]
Parallel research peer created: #14 `CARD-NEW-TOP10-RESEARCH-C0-EXECUTION-CAPACITY-AND-CONTROL-CUTOVER-PRECHECK`.

Dependency rule: B0 and C0 may research in parallel. C1 remains not admitted until B0 is accepted and a future `CandidateDecision → Canonical TrialSpec` handoff contract is pinned. Neither issue authorizes runtime, queue, runner, scheduler, or production mutation.
